priv/repo/migrations/20240701160107_add_operator_sign_ins.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Orbit.Repo.Migrations.AddOperatorSignIns do
+  use Ecto.Migration
+
+  def change do
+    execute(
+      "CREATE TYPE rail_line AS ENUM ('red', 'orange', 'blue')",
+      "DROP TYPE rail_line"
```

Comment:
**praise: ** I didn't know executes could take a rollback statement as the second argument. nice!

Reply:
Yes, it's quite handy! Lets you intersperse `execute` with other migration operations without having to then separate things out into a separate `up` and `down` just because of a single raw SQL statement you need.